task-management/src/main/webapp/vue-app/taskDrawer/components/TaskDrawer.vue

```diff
@@ -545,6 +545,7 @@
         this.showEditor=false;
         this.task={};
         document.dispatchEvent(new CustomEvent('drawerClosed'));
+        document.dispatchEvent(new CustomEvent('loadTaskLabels', {detail: {}}));
```

Comment:
Couldn't we reinitialize labels when opening the drawer ?

Reply:
> Couldn't we reinitialize labels when opening the drawer

This is what is already done and causing the problem. I used the same logic used when opening the drawer to reinitialize the labels field. 

> add the logic to the drawerClosed even

And I preferred calling an existing event `loadTaskLabels` that will do all the logic without adding it again in a new listener in the `labelsComponent`.